health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/AdvancedMapCharts/LatLongMap/LatLongMap.js

```
@@ -149,50 +182,73 @@ const LatLongMap = ({
           setFilterFeature({finalFilter: name})
         } else return;
       }
+
+      const boundaryLevelMap = Digit.SessionStorage.get("levelMap")
   
       if (level === 2) {
+        const boundaryLevel = getBoundaryTypeByLevel("level-two", boundaryLevelMap);
         let dummy = {...filterStack};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'dummy' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)